src/strand-dropdown/strand-dropdown.html

```diff
@@ -51,13 +52,18 @@
 						id="itemRecycler"
 						data="{{data}}"
 						hidden$="{{!_hideInsertionPoints(data)}}"
+						scope="{{scope}}"
```

Comment:
Consider changing this to `scope="{{ref}}" and getting rid of the `scope` property  on <strand-dropdown/>